src/pages/dev/axelarjs-sdk/tx-status-query-recovery.mdx

```diff
@@ -1,18 +1,30 @@
-# Query and recover GMP transactions
+# Programmatically query and recover GMP transactions
 
-Occasionally, transactions can get "stuck" in the pipeline from a source to destination chain (e.g. due to one-off issues that arise with relayers that operate on top of the network).
+import { Callout } from "/src/components/callout"
 
-The `AxelarGMPRecoveryAPI` module in the AxelarJS SDK can be used by your dApp to query the status of any General Message Passing (GMP) transaction (triggered by either `callContract` or `callContractWithToken`) on the gateway contract of a source chain and trigger a manual relay from source to destination if necessary. - The [GMP status tracker](/dev/general-message-passing/debug/transaction-recovery/) on Axelarscan makes use of this feature.
+Transactions can occasionally get stuck in the pipeline from a source to destination chain, mostly from one of the two following reasons:
 
-### Install the AxelarJS SDK module (`AxelarGMPRecoveryAPI`)
+- The transaction is not relayed from the source chain into the Axelar network for processing.
```

Comment:
we could maybe add some more info here too, like specify that the relay issue is a microservice issue. Could maybe ask axelarons for a recommended point we could add here to make this more informative